tests/draft4/additionalItems.json

```diff
@@ -40,6 +40,11 @@
         },
         "tests": [
             {
+                "description": "fewer items is valid",
```

Comment:
The spirit of this is redundant with the test below it for "no additional items present", but if you feel like it's worth having separate tests for "less than" than "equal" it might be nice to reword the descriptions in that way specifically.